ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -938,6 +969,15 @@ main()
     # Remove the 'fake-NIFTI' file created in fix_3_clean for high-pass filtering of the CIFTI (if it exists)
 	$FSLDIR/bin/imrm ${concatfmrihp}.ica/Atlas
  
+	#always delete things with too-generic names
+	rm -f ${concatfmrihp}.ica/filtered_func_data.nii.gz ${concatfmrihp}.ica/Atlas.dtseries.nii
+
+	#delete highpass intermediates
+	if [[ "$DeleteIntermediates" == "1" ]]
+	then
+		rm -f ${concatfmri}.nii.gz ${concatfmri}_hp${hp}.nii.gz ${concatfmri}_Atlas.dtseries.nii ${concatfmri}_Atlas_hp${hp}.dtseries.nii
```

Comment:
The usage statement says that `--delete-intermediates` will delete the high-pass filtered files, yet here we are deleting the concatenated NIFTI and CIFTI files (prior to HP) as well. That seems a reasonable thing to do if the option is set, but means that we should change the usage statement to be more precise.  Also, if we are deleting the other concatenated files, shouldn't we be deleting the _cleaned_ concatenated files as well, if this option is set?

Reply:
I'm not sure what the usage should say, I wasn't involved in deciding what gets deleted.  Feel free to edit it.

Reply:
We should not delete the concatenated cleaned file, as that gets used.

Reply:
Ok, the concatenated cleaned file is preserved, but I edited the usage statement in what I'll be pushing shortly.